packages/python/plotly/plotly/express/_core.py

```diff
@@ -305,11 +305,38 @@ def make_trace_kwargs(args, trace_spec, g, mapping_labels, sizeref):
             elif k == "locations":
                 result[k] = g[v]
                 mapping_labels[v_label] = "%{location}"
+            elif k == "values":
+                result[k] = g[v]
+                _label = "value" if v_label == "values" else v_label
+                mapping_labels[_label] = "%{value}"
+            elif k == "parents":
```

Comment:
if we want to remove parents and ids from the hover template we can do it here but we could also keep them

Reply:
This makes sense to me ATM.